image/templates/helm/stackrox-central/templates/02-config-controller-02-deployment.yaml

```diff
@@ -44,6 +45,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        {{- include "srox.envVars" (list . "deployment" "config-controller" "config-controller") | nindent 8 }}
```

Comment:
The fourth parameter needs to be the container name.

Reply:
700c37f